### ISSUE TITLE
Configure Federation with Slack

### ISSUE THREAD

**Issue by martindale** [FabricLabs/riot#2]
We have an existing Slack community as part of Maki, so we'll need to link the two services using the Matrix protocol.  Task is to configure the Slack bridge, then configure Synapse (our Matrix homeserver) to federate identity across both platforms.

**Comment by naterchrdsn** [comment]
this looks good: https://github.com/matrix-org/matrix-appservice-slack

**Comment by naterchrdsn** [comment]
do you have a server already for this?

**Comment by martindale** [comment]
@naterchrdsn yes; `matrix.fabric.fm` is the server, `chat.fabric.fm` is the client (running our `#fabric` branch with one blocking bug in #3)
